exercises/robot-simulator/src/main/java/GridPosition.java

```diff
@@ -0,0 +1,16 @@
+public final class GridPosition {
+
+    final int x;
+
+    final int y;
+
+    GridPosition(final int x, final int y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    boolean equals(final GridPosition gridPosition) {
```

Comment:
Can you talk about the decision to overload instead of override [`equals()`](https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#equals-java.lang.Object-)?  Not saying it's wrong, just want to talk it through.